### PR TITLE
Show Explore Button when signed In.

### LINKED ISSUE #587: [BUGS] "Sign Up" button still shows after user signs In.
### Description

I would like to work on the issue i described in the Title, where the button to sign up is still shown after user signs-up.

### Screenshots

_No response_

### Additional information

_No response_

### 🥦 Browser

Google Chrome

### 👀 Have you checked for similar open issues?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read the [Contributing Guidelines](https://github.com/IAmTamal/Milan/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by temidayo-falomo**
# Fixes Issue

**My PR closes #587**

## What did you do ?

---I wrote a ternary operator to handle the displaying of the separate buttons.

# Check List (Check all the applicable boxes)

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.

# Screenshots

Screenshot 2023-02-03 at 3 15 33 PM

### CODE REVIEW

File: src/components/Banners/Banner.jsx [hunk 1]
```diff
@@ -28,16 +29,29 @@ const Banner = () => {
...
+                {Cookies.get("token") || Cookies.get("club") ? (
+                  <button
+                    className="btn btn-warning banner_signup_btn"
+                    onClick={() => {
+                      nav("/shop");
+                    }}
+                    data-cy="landingpage-club-signup"
+                    id="landingpage-club-signup"
+                  >
+                    Exolore our brand new shop !
```
Comment (IAmTamal) [inline src/components/Banners/Banner.jsx:41]: There's a small typo here, do fix it

**Review by IAmTamal — APPROVED** [review]
All other changes  LGTM, howver from the next time please DONOT close the issues by yourself. Merging the PRs would close them automatically.

File: src/components/Banners/Banner.jsx (hunk 1 above)
Comment (temidayo-falomo) [reply src/components/Banners/Banner.jsx:41]: Fixing it rn.

**Review by IAmTamal — APPROVED** [review]
LGTM ✅
Thankyou @temidayo-falomo for making the project much better !

**Pull request merged**